gui/mainwindow.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Cppcheck - A tool for static C/C++ code analysis
- * Copyright (C) 2007-2020 Cppcheck team.
+ * Copyright (C) 2007-2021 Cppcheck team.
```

Comment:
For information.. these copyright years are updated during a release "automatically". you don't need to tweak these manually.